### PR TITLE
chore: tidy up the features.json for palazzo milestone

### PR DESCRIPTION
**Pull request by gordsport**
This archives off the cosmic features.json file and tidies up current one ready for the addition of more palazzo ACs

- The tidied features.json file will be used with this [PR](https://github.com/vegaprotocol/jenkins-shared-library/pull/703) to get metrics on coverage
- the cosmic-features.json is purely a record of what was there at this time. It could be removed when we are sure that we do not need the information

### CODE REVIEW

File: protocol/cosmic-features.json [hunk 1]
```diff
@@ -0,0 +1,927 @@
...
+      "0056-REWA-101",
+      "0056-REWA-102",
+      "0056-REWA-103",
+      "0056-REWA-104",
+      "0056-REWA-105",
+      "0056-REWA-106",
+      "0056-REWA-107",
+      "0056-REWA-108"
+    ]
+  },
```
Comment (cdummett) [inline protocol/cosmic-features.json:534]: Possibly these should be removed as they are disabled for cosmic?

Or do we include as they are implemented?

File: protocol/cosmic-features.json [hunk 2]
```diff
@@ -0,0 +1,927 @@
...
+    "milestone": "deployment-3",
+    "acs": [
+      "0028-GOVE-145",
+      "0028-GOVE-146",
+      "0028-GOVE-147",
+      "0028-GOVE-148",
+      "0028-GOVE-149"
+    ]
+  },
+  "Spot": {
```
Comment (cdummett) [inline protocol/cosmic-features.json:571]: Not sure this Spot section should be here.

File: protocol/cosmic-features.json [hunk 3]
```diff
@@ -0,0 +1,927 @@
...
+      "0074-BTCH-015",
+      "0074-BTCH-016",
+      "0074-BTCH-019",
+      "0079-TGAP-006",
+      "0079-TGAP-007",
+      "0081-SUCM-033",
+      "0081-SUCM-034"
+    ]
+  },
+  "Fixed Size Amendments": {
```
Comment (cdummett) [inline protocol/cosmic-features.json:915]: Not a cosmic feature, should probably be removed.

**Review by cdummett — COMMENTED** [review]
Few small comments re features which were deferred / disabled / not implemented for cosmic.

File: protocol/cosmic-features.json (hunk 1 above)
Comment (gordsport) [reply protocol/cosmic-features.json:534]: I thiink we should keep this in as we may add more Teams ACs under: https://github.com/vegaprotocol/specs/issues/1999

The feature will be enabled for palazzo milestone

File: protocol/cosmic-features.json (hunk 2 above)
Comment (gordsport) [reply protocol/cosmic-features.json:571]: I left this in so we had a record of all the spot ACs for when spot gets prioritised.

File: protocol/cosmic-features.json (hunk 3 above)
Comment (gordsport) [reply protocol/cosmic-features.json:915]: Happy to remove, I just archived the old features.json as it was so we have a record of how it looked.

**Review by cdummett — APPROVED** [review]

**Pull request merged**